README.md

```diff
@@ -110,86 +110,122 @@ bash test/test_cleanup.sh
 
 ## Usage
 
-Now that your virtual environment is set up and the workflows are deployed and
-tested, you can go ahead and run the workflows on your samples.
+Now that your virtual environment is set up and the workflow is deployed and
+tested, you can go ahead and run the workflow on your samples.
 
-But first, here is a brief description of what each of the three workflows
-does:
+But first, here is a brief description of what the workflow does:
 
 ### Workflow description
 
-The repository contains the following workflows, all implemented in Snakemake
-and fully containerized:
+The workflow consist on one main `Snakefile` 
+gathering three different subworkflows,
```

Comment:
I actually don't think that users need to know that there are three subworkflows. It just makes things appear more complicated. Just merge the workflow description into one. And while you are at it, move it out of the "Usage" section - it doesn't make sense to be there. If I want to know how to use something, I don't want to be lectured on how it works 😆

Reply:
Should the workflow description be before the installation? bellow the summary?  I think it is better than have it in the middle or the bottom.

Reply:
Both have advantages. Having it below the synopsis flows better, but people often want to get to the most important bits quickly, so synopsis > usage > description also makes sense. Up to you.